src/fr-webauthn/script-parser.ts

```diff
@@ -5,38 +5,43 @@ function parseWebAuthnRegisterText(text: string): PublicKeyCredentialCreationOpt
   const txtEncoder = new TextEncoder();
 
   // e.g. `attestation: "none"`
-  const attestation = getIndexOne(text.match(/attestation:\s{0,}"(\w+)"/)) as AttestationType;
+  const attestation = getIndexOne(text.match(/attestation"{0,}:\s{0,}"(\w+)"/)) as AttestationType;
```

Comment:
Duplicated the change needed for 6.5.2 to all parsed keys in case they change in future versions.